osf_tests/test_prereg.py

```diff
@@ -120,7 +120,7 @@ def test_has_project_and_draft_registration(self):
         )
 
         with override_switch(name=OSF_PREREGISTRATION, active=True):
-            prereg_schema = RegistrationSchema.objects.get(name='OSF Preregistration')
+            prereg_schema = RegistrationSchema.objects.filter(name='OSF Preregistration')[0]
```

Comment:
Would this be better to use `.get()` in the same way https://github.com/CenterForOpenScience/osf.io/pull/9313/files#diff-80b55329bb6b12783ad2f51570471379R13 does?